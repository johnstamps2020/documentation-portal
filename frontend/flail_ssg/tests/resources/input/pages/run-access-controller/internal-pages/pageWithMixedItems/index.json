{
  "$schema": "/frontend/page-schema.json",
  "title": "Page with mixed items",
  "template": "page",
  "class": "threeCards",
  "selector": {
    "label": "Select page",
    "selectedItem": "Page with mixed items",
    "items": [
      {
        "label": "Page with only internal docs",
        "page": "../pageWithOnlyInternalDocs"
      },
      {
        "label": "Page with only internal items",
        "page": "../pageWithOnlyInternalItems"
      }
    ]
  },
  "items": [
    {
      "label": "InsuranceSuite Configuration Upgrade Tools",
      "id": "isconfigupgradetoolsdraft"
    },
    {
      "label": "Feature Preview",
      "class": "categoryCard cardShadow",
      "env": ["int"],
      "items": [
        {
          "label": "Page with only internal docs",
          "page": "../pageWithOnlyInternalDocs",
          "env": ["int"]
        }
      ]
    }
  ]
}
