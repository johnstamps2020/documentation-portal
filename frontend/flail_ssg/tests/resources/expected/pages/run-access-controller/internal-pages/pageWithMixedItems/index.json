{
  "$schema": "/frontend/page-schema.json",
  "title": "Page with mixed items",
  "template": "page",
  "class": "threeCards",
  "selector": {
    "label": "Select page",
    "selectedItem": "Page with mixed items",
    "items": [
      {
        "label": "Page with only internal docs",
        "page": "../pageWithOnlyInternalDocs",
        "public": false,
        "internal": true
      },
      {
        "label": "Page with only internal items",
        "page": "../pageWithOnlyInternalItems",
        "public": false,
        "internal": true
      }
    ]
  },
  "items": [
    {
      "label": "InsuranceSuite Configuration Upgrade Tools",
      "id": "isconfigupgradetoolsdraft",
      "public": false,
      "internal": false
    },
    {
      "label": "Feature Preview",
      "class": "categoryCard cardShadow",
      "env": ["int"],
      "items": [
        {
          "label": "Page with only internal docs",
          "page": "../pageWithOnlyInternalDocs",
          "env": ["int"],
          "public": false,
          "internal": true
        }
      ]
    }
  ],
  "public": false,
  "internal": false
}
